docs/intelligentapps/finetune.md

```diff
@@ -0,0 +1,278 @@
+---
+Order: 5
+Area: intelligentapps
+TOCTitle: Finetune
+ContentId:
+PageTitle: Finetune AI Models
+DateApproved:
+MetaDescription: Use custom dataset to finetune a generative AI model in the Azure cloud or locally with GPUs. Deploy the finetuned model to the Azure cloud or download adapter file.
```

Comment:
What is an "adapter file"? This text shows up in Google/Bing - people will not know what this concept is about. Make sure to keep this text as clear as possible so people know it's relevant just from search results.

Reply:
updated